nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/groovy/org/apache/nifi/processors/standard/TestPutDatabaseRecord.groovy

```diff
@@ -301,6 +301,73 @@ class TestPutDatabaseRecord {
         conn.close()
     }
 
+    @Test
+    void testInsertNonRequiredColumns() throws InitializationException, ProcessException, SQLException, IOException {
```

Comment:
Do you want to add a test case to show SQLDataException being thrown?

Reply:
Yes it will take a badly-behaved PutDatabaseRecord subclass to achieve, but should add a test to cover the path regardless